doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-intro-con.adoc

```diff
@@ -5,7 +5,7 @@ Test scenarios in {PRODUCT} enable you to validate the functionality of business
 
 {PRODUCT} currently supports both the new *Test Scenarios* designer and the former *Test Scenarios (Legacy)* designer. The default designer is the new test scenarios designer, which supports testing of both rules and DMN models and provides an enhanced overall user experience with test scenarios. The new test scenarios designer currently can use only the default KIE session. If required, you can continue to use the legacy test scenarios designer, which supports rule-based test scenarios only.
 
-With the test scenarios designer, you execute all scenarios from the `.scesim` file at one time, whereas with the legacy test scenarios designer, you can execute test scenarios one at a time or as a group. The group execution contains all the scenarios from one package. Test scenarios are independent, so one scenario cannot affect or modify the other. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios.
+The defined test scenarios in both the designers can be executed in multiple ways, for example you can run the all the available test scenarios at a project level or inside a particular test scenario asset. For more information, see xref:test-designer-run-test-proc[]. Test scenarios are independent, so one scenario cannot affect or modify the other. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios.
```

Comment:
Do we use "execute" or "run" when referring to running test scenarios?

Number of ways is more common in the docs:
s/multiple ways/number of ways

Typo:
s/run the all the/run all the

s/modify the other./modify another./modify other test scenarios.

s/particular/specific/

I rewrote some of the paragraph, please feel free to adopt the parts you agree with:

You can execute the defined test scenarios in a number of ways, for example, you can run available test scenarios at the project level or inside a specific test scenario asset. Test scenarios are independent and cannot affect or modify other test scenarios. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios. For more information about running test scenarios, see xref:test-designer-run-test-proc[].

Reply:
Thank you for the detailed review. 
I will incorporate the typo and suggested changes.